tests/unit/picasso/test_rest.py

```diff
@@ -35,6 +39,25 @@
 NOT_EXIST_IMG_NAME = _cfg.tests.not_exist_img_name
 NOT_EXIST_SNAP_NAME = _cfg.tests.not_exist_snap_name
 
+processes = None
+
+
+def setUpModule():
+    global processes
+    p1 = Process(target=start_name_server)
+    p2 = Process(target=start_rpc_server)
+    p3 = Process(target=rest.start)
+    p1.start()
+    p2.start()
+    p3.start()
+    time.sleep(5)
```

Comment:
just a minor nit: you could start the processes the same way you are terminating itthem: `[p.start() for p in processes]`
You could leave it as it is if there are some other reasons.